qiskit_optimization/problems/quadratic_program.py

```diff
@@ -811,161 +799,58 @@ def maximize(self,
         self._objective = QuadraticObjective(self, constant, linear, quadratic,
                                              QuadraticObjective.Sense.MAXIMIZE)
 
-    def from_docplex(self, model: Model) -> None:
-        """Loads this quadratic program from a docplex model.
+    def from_model(self, model: Model, translator: Optional[ModelTranslator] = None) -> None:
+        """Loads this quadratic program from an optimization model.
```

Comment:
Is `model: Model` valid? I guess `Model` here comes from DOcplex? Should it be something more generic?

I suggest to make `from_model` a static method to be able to create a model like `qp = QuadraticProgram.from_model(another_model, translator)`

Reply:
You are right. `model: Model` is invalid here because `Model` is `docplex.mp.model.Model`.  I agree to making `from_model` static.